non-code.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Non-Code
+layout: default
+permalink: /non-code/
+---
+
+While the focus of this site is open source software, we recommend checking out [Creative Commons](http://creativecommons.org/choose/) for most other types of projects.
+
+If your project contains a mix of software and content, you can include multiple licenses, as long as you are explicit about which license applies to what segments of the project.  See [the license disclaimer for this site](https://github.com/github/choosealicense.com#license) as an example.
```

Comment:
> a big edge case for the site's purpose 

If the site to demystify licenses has a mixed license, it's probably worth mentioning, don't you think? Totally open to any ideas about how to make it less mysterious without going into a lot more detail.

> start an FAQ

What if we moved the "What if none of these work for me?" section of the homepage into an FAQ page, and added this question?